tiles-generation/cdk/lambda/event-process.js

```diff
@@ -41,11 +41,11 @@ const getStateInformation = function(event) {
             }
         }
         return stateInfo;
-    } catch(err) {
+    } catch (err) {
         console.log(err);
     }
```

Comment:
have we validated that these errors logged via console.log can be seen on the Cloudwatch?